src/layouts/components/Editor/extensions/Details/IsomerDetailGroup.ts

```diff
@@ -197,37 +210,13 @@ export const IsomerDetailsGroup = Node.create<IsomerDetailGroupOptions>({
           )
           .run()
       },
-      unsetDetailsGroup: () => ({
-        chain,
-        state,
-      }: {
-        state: EditorState
-        chain: ChainedCommands
-      }) => {
-        const { selection } = state
-        const currNode = findParentNode((t) => t.type === this.type)(selection)
-        if (!currNode) return false
-        const startPos = currNode.pos
-        const range = { from: startPos, to: startPos + currNode.node.nodeSize }
-
-        // TODO: check type
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        return chain()
-          .insertContentAt(range, [])
-          .setTextSelection(startPos + 1)
-          .run()
+      unsetDetailsGroup: () => ({ tr, editor }) => {
+        editor.state.selection.replace(tr)
```

Comment:
sanity check - this just replaces the selection with an empty string right? what happens if we have a selection within the accordion and we call this? would only the selection (ie, the text) be replaced? 

iirc, tiptap uses `toggleNode` or smth for their exported nodes, maybe that might be worth checking out

Reply:
You are right to call this out, there is an underlying assumption that this func is only called iff the entire accordion is selected. This is maintained (correct me if i am wrong here) since the delete is only called from pressing the button, which only appears on select of the entire accordion block. 


But out of curorsity, say this could actually be called when the selection is within the accordion, how will the `toggleNode` work ah? if > 1 accordion in the page, this toggle node will not work as expected right?

Reply:
the base node is `detailSummary`, which we can call [toggle node](https://tiptap.dev/docs/editor/api/commands/toggle-node) on. alternatively, we could also consider just using tiptap's default [unsetDetails](https://tiptap.dev/docs/editor/api/nodes/details#unset-details), which is less destructive.

Reply:
this was the original implementation! this had to be coded out so that we dont unset the details causing a backspace on a detail deleting the entire detail group into an inconsistent state that we do not expect